tests/Features/Macro.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use Illuminate\Support\Traits\Macroable;
+use PHPUnit\Framework\TestCase;
+
+uses(Macroable::class);
+
+beforeEach(function () {
```

Comment:
`beforeEach()->macro(...`.

Reply:
Oh yeah, even better. You see I'm pretty new. 😄 
https://github.com/pestphp/pest/pull/109/commits/9a0cfaf33940af11471c89147c56735ad721d494